### PR TITLE
Update data-protection.md

### PR DESCRIPTION
**Pull request by ekoner**
Aligned data protection advice with guidance given elsewhere on sharing personal data in contact fields.

### CODE REVIEW

File: documentation/data-protection.md [hunk 1]
```diff
@@ -6,8 +6,19 @@ The [Data Protection Act](https://www.gov.uk/data-protection/the-data-protection
...
 
-[Open data](https://theodi.org/what-is-open-data) should not contain personal or sensitive personal data that could allow a living person to be identified (there are [exceptions](https://www.gov.uk/government/publications/dft-senior-officials-business-expenses-and-meetings-april-to-june-2014) but these don't apply to [360Giving](http://www.threesixtygiving.org/)). Data published to the [360Giving standard](http://www.threesixtygiving.org/standard/) should be anonymised to protect privacy as outlined in the ICO's [anonymisation code of practice](https://ico.org.uk/for-organisations/guide-to-data-protection/anonymisation/).
+## Publishing personal data with 360Giving
+We encourage publishers to carefully consider the value of sharing any personal data as part of their 360Giving publication and discourage sharing of sensitive personal data.
```
Comment (caprenter) [inline documentation/data-protection.md]: "We ... discourage sharing of sensitive personal data." - except we include fields that expect personal data in the standard?. I'd suggest something along the lines of:
"We encourage publishers to carefully consider the value of sharing any personal data as part of their 360Giving publication and exercise care when populating fields in the data standard that may expose sensitive personal data. "

File: documentation/data-protection.md [hunk 2]
```diff
@@ -6,8 +6,19 @@ The [Data Protection Act](https://www.gov.uk/data-protection/the-data-protection
...
+## Publishing personal data with 360Giving
+We encourage publishers to carefully consider the value of sharing any personal data as part of their 360Giving publication and discourage sharing of sensitive personal data. 
+
+Before making a decision on including personal data:
+
+ 1. Review the ICO's [Key definitions of the data protection act](https://ico.org.uk/for-organisations/guide-to-data-protection/key-definitions/) to understand the difference between non-personal data, personal data and sensitive personal data;
+ 2. Review the ICO's [Guide to data protection](https://ico.org.uk/for-organisations/guide-to-data-protection/) to understand your organisation's obligations;
+ 3. Ensure your organisation has the [power to share the data](https://ico.org.uk/for-organisations/guide-to-data-protection/data-sharing/); 
+ 4. Restrict personal data to names used in official capacity, for example the contact names at the funding and recipient organisations, names of recipients of funds e.g. scholarships and identifiers that are open or in the public domain, for example [orcid](https://orcid.org/);
```
Comment (caprenter) [inline documentation/data-protection.md]: I'd suggest breaking this into another section:

"Once you have decided to publish personal data:
1. Restrict..."

File: documentation/data-protection.md (hunk 1 above)
Comment (ekoner) [inline documentation/data-protection.md]: Sensitive personal data isn't the same as personal data, think medical information, religious affiliation etc

File: documentation/data-protection.md (hunk 2 above)
Comment (ekoner) [inline documentation/data-protection.md]: Done

File: documentation/data-protection.md (hunk 1 above)
Comment (caprenter) [inline documentation/data-protection.md]: Ah, yes - good point.

**Comment by ekoner** [comment]
Ping @caprenter - can you review new changes?

**Pull request merged**